### PR TITLE
zh: remove contents about DM-portal

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- remove contents about DM-portal because we will not maintain it recently.

NOTE: keep contents in v1.0 so that the old user can still ref it.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [x] Have version specific changes 
- [x] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GMHDBJD @ireneontheway PTAL

**Comment by csuzhangxc** [comment]
@TomShawn two LGTM now, PTAL

**Comment by csuzhangxc** [comment]
> @csuzhangxc Should we add an alias for this deleted file in another document?

I think we can just delete it permanently.

**Comment by TomShawn** [comment]
> > @csuzhangxc Should we add an alias for this deleted file in another document?
> 
> I think we can just delete it permanently.

If we just directly delete it, users who kept the document link may find the link broken. I think we'd better redirect the document link to another document address to avoid the 404 error. To do it, let's find another appropriate document to be redirected at?

**Comment by csuzhangxc** [comment]
> > > @csuzhangxc Should we add an alias for this deleted file in another document?
> > 
> > 
> > I think we can just delete it permanently.
> 
> If we just directly delete it, users who kept the document link may find the link broken. I think we'd better redirect the document link to another document address to avoid the 404 error. To do it, let's find another appropriate document to be redirected at?

OK, I added an alias in https://github.com/pingcap/docs-dm/pull/232/commits/5e5b226f466a91c43baae73d64d9039f84be7537

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: DM 任务完整配置文件介绍
-aliases: ['/docs-cn/tidb-data-migration/dev/task-configuration-file-full/']
+aliases: ['/docs-cn/tidb-data-migration/dev/task-configuration-file-full/', '/zh/tidb-data-migration/dev/dm-portal']
```
Comment (TomShawn) [inline zh/task-configuration-file-full.md:3]:
```suggestion
aliases: ['/docs-cn/tidb-data-migration/dev/task-configuration-file-full/','/zh/tidb-data-migration/dev/dm-portal']
```

**Pull request merged**